flyteorg/furo
# Catchup to upstream

This PR merges changes from the upstream https://github.com/pradyunsg/furo repo and applies our customizations for Flyte on top of it.

I did this by:
- doing a git merge on the upstream main branch
- resolving conflicts
- updating parts of the codebase to support custom Flyte repo behavior and aesthetics

## QA

Check out the updated theme in these test PRs:
- https://github.com/flyteorg/flyte/pull/2517 - [Docs Preview](https://flyte--2517.org.readthedocs.build/en/2517/getting_started/index.html)
- https://github.com/flyteorg/flytesnacks/pull/759 - [Docs Preview](https://flyte--759.org.readthedocs.build/projects/cookbook/en/759/integrations.html)
